symfit/core/support.py

```diff
@@ -21,8 +20,15 @@
 
 if sys.version_info >= (3,0):
     import inspect as inspect_sig
+    from functools import wraps
 else:
     import funcsigs as inspect_sig
+    from functools32 import wraps
+
+if sys.version_info >= (3, 5):
+    from functools import partial
+else:
+    from ._support import repeatable_partial as partial
```

Comment:
The file _support needs to be renamed if it's just this partial hack. Currently it's hard to track what's coming from where.

Reply:
Perhaps it could be named `_repeatable_partial.py` instead? The reason for having it separate is such that `repeatable_partial` is only ever defined in relevant python versions.

Reply:
That works for me.